build-android/build_all.sh

```diff
@@ -41,7 +41,7 @@ function findtool() {
 # Check for dependencies
 findtool aapt
 findtool zipalign
-findtool jarsigner
```

Comment:
I think this is eventually necessary, and it was easier to do this than install the JDK locally for me.